walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,93 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.*;
+import java.util.Comparator;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+  public ArrayList<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
Usually you'd try not to change the method definition when you are just trying to implement it. Can you leave this as Collection<TimeRange>?

Reply:
Collections.sort was not working that is why I changed the method definition.